tests/test_portchannel_global.rb

```diff
@@ -37,6 +38,11 @@ def cleanup
     config_no_warn "no port-channel load-balance #{ethernet}"
   end
 
+  def n3k_is_t2?
+    return unless /N3/ =~ node.product_id
+    Platform.chassis['descr'].include?('Nexus 31') ? true : false
```

Comment:
No. include? is in the strings as well. I used it before many times.
check this:
https://ruby-doc.org/core-2.2.0/String.html#method-i-include-3F.

But I did not know about 31xx being Nexus3000 C3132Q. 
Any suggestions on what to look for, to determinte if it is T2 chassis?